R/setConfig.R

```diff
@@ -37,9 +37,9 @@
 #' @param outputfolder The folder all outputs should be written to. In the
 #' default case this argument is set to NA meaning that the default folder
 #' should be used which is <mainfolder>/output
-#' @param bundlefolder The folder all output bundle should be written to. In the
-#' default case this argument is set to NA meaning that the default folder
-#' should be used which is <mainfolder>/bundle
+#' @param pucfolder The folder all portable unaggregated collections (puc) files
+#' should be written to. In the default case this argument is set to NA meaning
+#' that the default folder should be used which is <mainfolder>/puc
```

Comment:
They are not only written to that location, but also read from there, right?
```suggestion
#' @param pucfolder The path where portable unaggregated collection (puc) files
#' are located. NA by default, which means <mainfolder>/puc
```